src/sql/strong-oracle/create.sql

```diff
@@ -0,0 +1,332 @@
+BEGIN
```

Comment:
This is from the top-voted answer on stackoverflow: http://stackoverflow.com/questions/1799128/oracle-if-table-exists